packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -446,85 +447,103 @@ const AccountSwitcher = ({
+                        <Text line_height='s' size='xs' weight='bold' color='prominent'>
+                            <Localize i18n_default_text='Total assets' />
+                        </Text>
+                        <Text size='xs' color='prominent' className='acc-switcher__balance'>
+                            <Money
+                                currency={isRealAccountTab ? account_total_balance_currency : vrtc_currency}
+                                amount={formatMoney(
+                                    isRealAccountTab ? account_total_balance_currency : vrtc_currency,
+                                    isRealAccountTab ? getTotalRealAssets() : getTotalDemoAssets(),
+                                    true
+                                )}
+                                show_currency
+                                should_format={false}
+                            />
+                        </Text>
+                    </div>
+                    <Text color='less-prominent' line_height='xs' size='xxxs' className='acc-switcher__total-subtitle'>
+                        {localize('Total assets in your Deriv accounts.')}
                     </Text>
-                    <Icon icon='IcLogout' className='acc-switcher__logout-icon drawer__icon' onClick={handleLogout} />
-                </div>
-            </div>
+                    <div className='acc-switcher__separator' />
+
+                    <TradersHubRedirect />
+
+                    <div className='acc-switcher__footer'>
+                        {isRealAccountTab && has_active_real_account && !is_virtual && (
+                            <Button
+                                className='acc-switcher__btn--traders_hub'
+                                secondary
+                                onClick={
+                                    has_any_real_account && !hasSetCurrency
+                                        ? setAccountCurrency
+                                        : () => openRealAccountSignup('manage')
+                                }
+                            >
+                                {localize('Manage accounts')}
+                            </Button>
+                        )}
+                        <div id='dt_logout_button' className='acc-switcher__logout' onClick={handleLogout}>
+                            <Text color='prominent' size='xs' align='left' className='acc-switcher__logout-text'>
+                                {localize('Log out')}
+                            </Text>
+                            <Icon
+                                icon='IcLogout'
+                                className='acc-switcher__logout-icon drawer__icon'
+                                onClick={handleLogout}
+                            />
+                        </div>
+                    </div>
+                </React.Fragment>
+            ) : (
+                <Loading is_fullscreen={false} />
```

Comment:
@amina-deriv can we just return <Loading /> without wrapping the whole code block inside is_landing_company_loaded condition?

Reply:
@Matin As per card requirement, we need to show a loader when data is not completely fetched.